tap_jira/__init__.py

```diff
@@ -93,6 +93,9 @@ def sync():
     # data for the second stream, but the second stream hasn't output its
     # schema yet
     for stream in streams_.ALL_STREAMS:
+        # Update pk for users stream to key for on prem jira instance
+        if stream.tap_stream_id == "users" and Context.client.is_on_prem_instance:
+            stream.pk_fields = ["key"]
```

Comment:
We already do this in `generate_metadata()`, can we refactor so we only have 1 place in code in which we generate our schema?

Reply:
Done